web/themes/custom/cloudy/pattern-lab/_patterns/00-elements/02-typography/25-lists.twig

```diff
@@ -0,0 +1,38 @@
+<h3>Unordered List</h3>
+<ul>
+  <li>This is a list item in an unordered list</li>
+  <li>An unordered list is a list in which the sequence of items is not important. Sometimes, an unordered list is a bulleted list. And this is a long list item in an unordered list that can wrap onto a new line. </li>
+  <li>
+    Lists can be nested inside of each other
+    <ul>
+      <li>This is a nested list item</li>
+      <li>This is another nested list item in an unordered list</li>
+    </ul>
+  </li>
+  <li>This is the last list item</li>
+</ul>
+
+<h3>Ordered List</h3>
+<ol>
+  <li>This is a list item in an ordered list</li>
+  <li>An ordered list is a list in which the sequence of items is important. An ordered list does not necessarily contain sequence characters.</li>
+  <li>
+    Lists can be nested inside of each other
+    <ol>
+      <li>This is a nested list item</li>
+      <li>This is another nested list item in an ordered list</li>
+    </ol>
+  </li>
+  <li>This is the last list item</li>
+</ol>
+
+<h3>Definition List</h3>
+<dt>Definition List Title</dt>
```

Comment:
Doesn't a definition title `<dt>` and definition item `<dd>` need to be wrapped in a definition list `<dl>`?